### ISSUE TITLE
LOTT: torches are visually broken

### ISSUE THREAD

**Issue by FinishedFragment** [minetest-LOTR/Lord-of-the-Test#144]
torches are looking strange

![normal torch](https://user-images.githubusercontent.com/44732327/55647051-34113a80-57cc-11e9-9a32-0db6909e086f.png)
![orc torch](https://user-images.githubusercontent.com/44732327/55647052-34a9d100-57cc-11e9-8dbe-6e8cefa3adc3.png)

but they are emitting light correctly

**Comment by ElonSatoshi** [comment]
which version?

**Comment by Amaz1** [comment]
No idea how to fix this, I think it's an engine bug. I'll change things back to 2d until the engine is fixed.

**Comment by Amaz1** [comment]
If/when this is fixed in the engine, I'll revert d37b770
Keeping this open till then as a reminder.

**Comment by kbpickens** [comment]
I've created a fix for this in my [repo](https://github.com/kbpickens/mt_lott_torches).  It creates Orc, Elf, and Human (basically default:torch, but it does not override default:torch) torches by replicating minetest_game's default:torch code, texture, and model and colorizing the nodes.  I haven't seen any buildings created since I cloned the repo this morning (6 Jan 2020), but in 1.1.0 (downloaded 24 Nov 2019), the wall-mounted torches placed by lottmapgen stuck out horizontally.

These are angled out like minetest_game torches, not flat against the wall like the older LOTT torches.
